app/controllers/jobs_controller.rb

```diff
@@ -37,7 +37,7 @@ def ensure_business_user!
 	end
 
 	def ensure_broker!
-		return if broker_signed_in? || current_user.business_user?
+		return if broker_signed_in? || current_user.class.name === "BusinessUser"
```

Comment:
current_user.class.name === "BusinessUser" should be:
current_user.class.name == "BusinessUser"

Double equals to in ruby, triple in javascript :stuck_out_tongue: @rabingaire